content/examples/basic/front-matter/en.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Front Matter
 weight: 14
+date: "2023-08-01"
+meta: 
```

Comment:
I'd feel happier reviewing this with a couple of test examples on this page that /weren't/ the special-cased `readingTime` metadata field or the top-level `date` field.

Please could you add a couple of test metadata fields that show off the rendering capabilities (or limits we need to be aware of) when using this feature?

```suggestion
meta: 
- type: Test Type 1!
  value: A test value
- type: test `type` 2?
  value: Another test value
```

Reply:
(examples are deliberately in reverse sort order, so any sorting imposed by the code is visible)

Reply:
for now we only have date and the meta item ReadingTime. In an earlier design we had more types so Erik adjusted the existing html partial and removed the types we don't have but we kept the partial in case we want to add more meta later.
We can list the available types we have now in the example page which for now is just Reading Time.

Peter is also working on the redesign of the whole content header which he will finish today and we want to start working on that in the afternoon. In the new header design everything comes to together: authors, meta (date + reading time), tags. So the styling of the meta Erik made for this PR is a start but will be finalized when we have the new header design in a separate story and pr

Reply:
@jpluscplusm I've made the example more clear that for now we only have type 'readingTime' for meta.

Reply:
@JorindeUsMedia My apologies - I'd misinterpreted https://github.com/cue-lang/cuelang.org/pull/383/files#diff-f9a6e91a7670ffb79caf08157f4d7619ed95c4c74f4e89994bdaa0f1e3c58bb0R5 as being an override for `readingTime`, but which caused any /other/ `meta.` type+value to be rendered unchanged. Thanks for the correction!

Reply:
@jpluscplusm I understand the confusion :) The old meta code from the earlier design was not in use anymore so for readingTime we took the old code and adjusted it a bit.